docs/content/guides/recipes/Terraform/howto-private-registry/index.md

```diff
@@ -0,0 +1,79 @@
+weight: 500
+categories: "How-To"
+tags: ["recipes", "terraform"]
+---
+
+This how-to guide will describe how to:
+
+- Configure a Radius Environment to be able to pull Terraform Recipe templates from a private git repository.
+
+### Prerequisites
+
+Before you get started, you'll need to make sure you have the following tools and resources:
+
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Radius Bicep VSCode extension]({{< ref "installation#step-2-install-the-vs-code-extension" >}})
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Radius initialized with `rad init`]({{< ref howto-environment >}})
+
+## Step 1: Create a personal access token
+
+Create a personal access token, this can be from [GitHub](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#about-personal-access-tokens), [GitLab](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html), [Azure DevOps](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows) or any other Git platform.
+
+The PAT should have access to read the files inside the specific private repository.
+
+## Step 2: Define a secret store resource
+
+Begin by configuring a [Radius Secret Store]({{< ref secretstore >}}) with the personal access token or username + password you previously created, which has access to your private git repository.
+
+Create a Bicep file `env.bicep`, import Radius, and  define your resource:
+
+{{< rad file="snippets/env.bicep" embed=true marker="//SECRETSTORE" >}}
+
+> Note the property `pat` is a required key that refers to your personal access token, while `username` is an optional key you can specify if your git platform requires a username.
+
+## Step 3: Configure Terraform Recipe git authentication
+
+`recipeConfig` allows you to configure how Recipes should be setup and run. One available option is to specify git credentials for pulling Terraform Recipes from git sources. For more information refer to the [Radius Environment schema]({{< ref environment-schema >}}) page.
+
+In your `env.bicep` file add an Environment resource, along with Recipe configuration which leverages the previously defined secret store for git authentication.
+
+{{< rad file="snippets/env.bicep" embed=true marker="//ENV" >}}
+
+## Step 4: Add a Terraform Recipe
+
+Update your Environment with a Terraform Recipe, pointing to your private git repository. Note that your `templatePath` should contain a `git::` prefix, per the [Terraform module documentation](https://developer.hashicorp.com/terraform/language/modules/sources#generic-git-repository).
+
+{{< rad file="snippets/env-complete.bicep" embed=true marker="//ENV" markdownConfig="{linenos=table,hl_lines=[\"23-30\"],linenostart=30}" >}}
```

Comment:
Could you double check this is highlighting correctly? On my Windows machine there needs to be one more semicolon highlighted, but I've seen line ending issues before on Unix/Windows so it might just be me.
<img width="593" alt="image" src="https://github.com/radius-project/docs/assets/54363786/5cc006e1-36e5-49eb-b4af-b0db59cca772">